docs/UserGuide.md

```diff
@@ -140,9 +141,10 @@ The `Tool Bar` at the top provides a set of configuration options that control t
 * `Breakdown by file type` : toggles the contribution bar to either display the bar by :
     * the total lines of codes added (if checkbox is left unchecked), or
     * a breakdown of the number of lines of codes added to each file type (if checkbox is checked). If sorting criteria is contribution, only lines contributed to selected file types will be taken into account.
+*`Merge group` : merges all the ramp charts of the same group together to form a single ramp chart.
```

Comment:
not properly formatted. 
Merges all the ramp charts of each group into a single ramp chart; aggregates the contribution of each group.